frontend/src/organisms/NewWorkoutForm.js

```diff
@@ -112,7 +111,7 @@ export function NewWorkoutForm({
                   (workout.length *
                     ((values.intervalLength || 0) +
                       (values.intervalPauseLength || 0))) -
-                  (values.intervalPauseLength || 0),
+                  (values.intervalPauseLength || 0) * (0 || currentList.length > 1),
```

Comment:
klidně ten počet vytáhni do nějaké separe funkce, ale tohle je dost nechutný. stačí ti spočítat to číslo, a když je menší než 0 tak vrátit nulu. tohle je strašná magie